components/brave_wallet_ui/components/shared/authorize-hardware-device/authorize-hardware-device.tsx

```diff
@@ -6,11 +6,17 @@
 import * as React from 'react'
 import { StyledIFrame } from './style'
 import { LEDGER_BRIDGE_URL } from '../../../common/hardware/ledgerjs/ledger-messages'
+import { BraveWallet } from '../../../constants/types'
 
-export const AuthorizeHardwareDeviceIFrame = () => {
-  const src = LEDGER_BRIDGE_URL + `?targetUrl=${encodeURIComponent(window.origin)}`
+export interface Props {
+  coinType: BraveWallet.CoinType
+}
+
+export const AuthorizeHardwareDeviceIFrame = (props: Props) => {
+  const { coinType } = props
+  const src = LEDGER_BRIDGE_URL + `?targetUrl=${encodeURIComponent(window.origin)}` + '&coinType=' + coinType
```

Comment:
We should be okay since these will always be numbers https://github.com/brave/brave-core/blob/53f188fbf5092ee884e26901507cd975e5d0bf15/components/brave_wallet/common/brave_wallet.mojom#L759-L763